light_node/src/main.rs

```diff
@@ -240,27 +243,45 @@ fn block_on_actors(
     ));
 
     // create pool for ffi protocol runner connections (used just for readonly context)
-    let tezos_readonly_api_pool = Arc::new(create_tezos_readonly_api_pool(
+    let tezos_readonly_api_pool = match create_tezos_readonly_api_pool(
         "tezos_readonly_api_pool",
         env.ffi.tezos_readonly_api_pool.clone(),
         &env,
         tezos_env.clone(),
         log.clone(),
-    ));
-    let tezos_readonly_prevalidation_api_pool = Arc::new(create_tezos_readonly_api_pool(
+    ) {
+        Ok(pool) => Arc::new(pool),
+        Err(e) => shutdown_and_exit!(
```

Comment:
@akoptelov 
please, rebase on actual develop,
because this is changed a litttle bit in actual develop,
shutdown_and_exit is removed and actor system is create when all is ok